.github/workflows/ci.yaml

```diff
@@ -46,7 +46,12 @@ jobs:
       - run: python -c 'import os; print(os.getenv("GOOGLE_APPLICATION_CREDENTIALS_JSON", "").strip())' > ${{ env.GOOGLE_APPLICATION_CREDENTIALS }}
       - run: sqlite3 /tmp/sqlite_default.db "VACUUM;"
       - run: pip3 install nox
-      - run: nox -s test_examples_by_dependency
+      - run: nox -s test_examples_by_dependency -- --cov=./ --cov-report=xml --cov-branch
```

Comment:
how is this change related to utils.load module?

Reply:
No - that was a bonus improvement, I noticed these tests were not contributing to the test coverage - which affected the coverage of `astro.utils.dependencies`. I could make a separate PR for this, would you prefer?

Reply:
https://github.com/astro-projects/astro/pull/243 -- should ideally take care of this too 🤞